index.bs

```diff
@@ -144,7 +145,7 @@ A <dfn>background fetch</dfn> consists of:
     * The UI may display |bgFetch|'s [=background fetch/title=].
     * The UI may select an icon for display from |bgFetch|'s [=background fetch/icons=], and [=/fetch=] it using a new [=/request=] with the following properties:
       : [=request/URL=]
-      :: The {{ImageResource}}'s {{ImageResource/src}}.
+      :: The {{ImageResource}}'s [=ImageResource/src=].
```

Comment:
Does this need to be [processed](https://w3c.github.io/image-resource/#dfn-process-an-imageresource-from-an-api) first?

Also, it seems weird that we can't use the fetching mechanism defined by the spec https://w3c.github.io/image-resource/#fetching-an-image-resource.

Reply:
Depends on your use-case. You weren't processing the ImageResource before, I was keeping the behaviour the same. Happy to update it if you think it makes sense here (I think it does).

> Also, it seems weird that we can't use the fetching mechanism defined by the spec.

It could, if you're OK with both changes I can include a fix for both.

Reply:
Seems like we should benefit from the processing now it's defined.

I commented on the image resource spec that the fetching didn't seem quite right, so I'm happy for that to be done in a later PR.

Reply:
I added the processing step. As for the fetching, I replied to your comment on the PR.